main.js

```diff
@@ -4,22 +4,34 @@ const active = require('./lib/active');
 const get = require('./lib/get');
 const dynamos = require('./lib/dynamos');
 const health = require('./lib/health');
+const cache = require('./lib/cache');
 
 let metrics;
 
 exports.health = health.check;
 
 exports.get = fromURL => {
+	if(process.env.USE_CACHE){
```

Comment:
yes - but was hoping to use an env var to make it easier to disable if it causes problems - also to switch between different caches.